Sources/Plasma/FeatureLib/pfGameGUIMgr/pfGameGUIMgr.cpp

```diff
@@ -738,34 +740,39 @@ bool    pfGameUIInputInterface::InterpretInputEvent( plInputEventMsg *pMsg )
         return handled || modal; // we "handle" it if we are modal, even if it didn't do anything
     }
 
-    plMouseEventMsg *pMouseMsg = plMouseEventMsg::ConvertNoRef( pMsg );
-    if( pMouseMsg && fManager->IsClickEnabled() )
-    {
-        if( pMouseMsg->GetButton() == kLeftButtonDown )
-        {
-            handled = fGUIManager->IHandleMouse( pfGameGUIMgr::kMouseDown, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), fModifiers, &fCurrentCursor );
+    plMouseEventMsg* pMouseMsg = plMouseEventMsg::ConvertNoRef(pMsg);
+    if (pMouseMsg && fManager->IsClickEnabled()) {
+        if (pMouseMsg->GetButton() == kLeftButtonDown) {
+            handled = fGUIManager->IHandleMouse(pfGameGUIMgr::kMouseDown, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), 0.f, fModifiers, &fCurrentCursor);
             if (handled)
                 fButtonState |= kLeftButtonDown;
         }
-        else if( pMouseMsg->GetButton() == kLeftButtonUp )
-        {
-            handled = fGUIManager->IHandleMouse( pfGameGUIMgr::kMouseUp, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), fModifiers, &fCurrentCursor );
+        else if (pMouseMsg->GetButton() == kLeftButtonUp) {
+            handled = fGUIManager->IHandleMouse(pfGameGUIMgr::kMouseUp, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), 0.f, fModifiers, &fCurrentCursor);
             if ((handled) || (fButtonState & kLeftButtonDown)) // even if we didn't handle the mouse up, if we think the button is still down, we should clear our flag
                 fButtonState &= ~kLeftButtonDown;
         }
-        else if( pMouseMsg->GetButton() == kLeftButtonDblClk )
-            handled = fGUIManager->IHandleMouse( pfGameGUIMgr::kMouseDblClick, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), fModifiers, &fCurrentCursor );
-        else if( fButtonState & kLeftButtonDown )
-            handled = fGUIManager->IHandleMouse( pfGameGUIMgr::kMouseDrag, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), fModifiers, &fCurrentCursor );
+        else if (pMouseMsg->GetButton() == kLeftButtonDblClk)
+            handled = fGUIManager->IHandleMouse(pfGameGUIMgr::kMouseDblClick, pMouseMsg->GetXPos(), pMouseMsg->GetYPos(), 0.f, fModifiers, &fCurrentCursor);
+        else if (pMouseMsg->GetButton() == kWheelPos || pMouseMsg->GetButton() == kWheelNeg) {
```

Comment:
Since this branch has a curly brace, we should give each branch of this if/else a curly brace.